xbmc/interfaces/legacy/ModuleXbmcplugin.h

```diff
@@ -91,21 +91,60 @@ namespace XBMCAddon
     void setResolvedUrl(int handle, bool succeeded, const XBMCAddon::xbmcgui::ListItem* listitem);
 
     /**
-     * addSortMethod(handle, sortMethod, label2Mask) -- Adds a sorting method for the media list.
+     * addSortMethod(handle, sortMethod  [,label2Mask]) -- Adds a sorting method for the media list.
      * 
      * handle      : integer - handle the plugin was started with.\n
-     * sortMethod  : integer - number for sortmethod see SortFileItem.h.\n
-     * label2Mask  : [opt] string - the label mask to use for the second label.  Defaults to '%D'
-     *               - applies to:
-     *                           - SORT_METHOD_NONE, SORT_METHOD_UNSORTED, SORT_METHOD_VIDEO_TITLE,
-     *                           - SORT_METHOD_TRACKNUM, SORT_METHOD_FILE, SORT_METHOD_TITLE,
-     *                           - SORT_METHOD_TITLE_IGNORE_THE, SORT_METHOD_LABEL,
-     *                           - SORT_METHOD_LABEL_IGNORE_THE, SORT_METHOD_VIDEO_SORT_TITLE,
-     *                           - SORT_METHOD_VIDEO_SORT_TITLE_IGNORE_THE, SORT_METHOD_FULLPATH,
-     *                           - SORT_METHOD_LABEL_IGNORE_FOLDERS, SORT_METHOD_CHANNEL
+     * sortMethod  : string - sortmethod see SortFileItem.h.\n
```

Comment:
in your python code, you pass 'xbmcplugin.SORTMETHOD_DATEADDED' to the addSortMethod function.
'xbmcplugin.SORTMETHOD_DATEADDED' is not an integer, it's a string.
the fact that kodi internally matches this to an int is not relevant for our python documentation imo.